src/DockMonitor.js

```diff
@@ -34,6 +35,10 @@ export default class DockMonitor extends Component {
     super(props);
     this.handleKeyDown = this.handleKeyDown.bind(this);
     this.handleSizeChange = this.handleSizeChange.bind(this);
+    this.parsedKeys = {
```

Comment:
Let's parse them in the event handler instead.
Feels wrong to remember something based on props.
They're not likely to change in our case, but still.